src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -176,6 +183,75 @@ export const checkNewBoundaryEnabled = async (
   return next()
 }
 
+/**
+ * Scan attachments on quarantine bucket and retrieve attachments from the clean bucket.
+ */
```

Comment:
For this PR, we are only scanning, not retrieving :) could add a TODO to indicate that we are not ready to turn on virus scanner flag yet

Reply:
Added comment in [44342cf](https://github.com/opengovsg/FormSG/pull/6734/commits/44342cff6f3614763f5e6fe34c1a735f650f9edf)! I'm keeping the comment to say retrieving because that's what the middleware should do, and instead added another line to say it's not completely implemented yet.